### DISCUSSION TITLE
Html practica

### DISCUSSION THREAD

**Discussion by AngelArielGOOD** [Launch-X-Latam/MisionFrontEnd#185] [Q&A]
Alguien sabe que es de practica en html?

**Comment by eldhaim** [comment]
https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/02%20-%20HTML/practicas :)

**Reply by AngelArielGOOD** [reply]
Gracias, y sabes si de la primera practica tambien hay que hacer repositorio o solo era de prueba las primeras practicas?

**Reply by FaraN1ght** [reply]
Hola Ángel, todas las prácticas deben documentarse puesto que en algún momento (no se sabe cuándo) pondrán un formulario para subir los repositorios por lo que es mejor tenerlas hechas con antelación. Adicional, el objetivo del bootcamp es aprender de todas las etapas del desarrollo web, por lo que en las prácticas pones en evidencia lo que aprendiste, además de que te servirán en la vida laboral cuando un empleador revise tu repositorio.

Ten en cuenta que estas preguntas ya las han respondido antes y también las hacen en los videos para que estés pendiente de más información. Mucha suerte!
